### ISSUE TITLE
What method is needed to drop fps to increase video length?

### ISSUE THREAD

**Issue by DuckersMcQuack** [kijai/ComfyUI-CogVideoXWrapper#210]
Or can the video only last for 2 seconds? As i've tried 12 fps and 24 fps, both times it's the same duration.

**Comment by cheezecrisp** [comment]
2 ways to extend the length of the video:
a) reduce the fps parameter in videocombine node
b) use frame interpolation to increase the number of frames  in the video
both of these methods slow down the motion.

**Comment by DuckersMcQuack** [comment]
Ah ok,  so model is strictly intended for 2 second videos? can't do half fps to last twice as long for instance?

**Comment by cheezecrisp** [comment]
> Ah ok, so model is strictly intended for 2 second videos? can't do half fps to last twice as long for instance?

I don't think there is a way to do that. But you can use the last frame in the video to generate another video and then concatenate them together.

**Comment by DuckersMcQuack** [comment]
> > Ah ok, so model is strictly intended for 2 second videos? can't do half fps to last twice as long for instance?
> 
> I don't think there is a way to do that. But you can use the last frame in the video to generate another video and then concatenate them together.

True. I need to see if it will result like animatediff models where last frame of the first video with the same seed will result in a X amount of second "still image video" lol

**Comment by zazoum-art** [comment]
Don't use the same seed. It works as cheezecrisp said. I have 1 minute and 39 secs so far with a fluid anime sequence. I told you the process in the other thread. It just needs a bit of color correction between the segments.

**Comment by zazoum-art** [comment]
[i2v.json](https://github.com/user-attachments/files/17695925/i2v.json)

Here is the workflow. With no surprise it is the basic workflow with a bit of help of Flowrense-2.
I can't post the whole sequence here cause the github limit is 10MB. Give me a few moments to  cut 2 characteristic raw low-res un-enchanched unedited sequences (not properly edited in a professional film direction manner).

**Comment by zazoum-art** [comment]
https://github.com/user-attachments/assets/782b6ef4-dbca-4601-8ee2-34957744681e

Here's an Enki-bilal dystopian style shot unedited. It is 22 secs and that's 4 clips of cogvideo, not 22. It is static camera following the woman, cause well, thats enki bilal.

Give me a few moments to cut a dynamic movement one.

**Comment by zazoum-art** [comment]
Uploading warrior_00079_thm2_apo8_iris3_iris2.mp4…

Here's a more dramatic one. 17secs

Of course you can't present these on festivals as they are. You have to edit the momentum and change the RGB properties. I use Premiere Pro. But as you can see the transitions are fluid.

The ugly faces, the upscaling, the enchancement, blurriness, sharpness etc. are easily dealt with Topaz Video AI. They can be converted to crystal clear 8K resolution videos with all distortions fixed in under 2 minutes of 4090 process I think.

**Comment by KrakeyMTL** [comment]
FYI by default both the video and the img2vid are 6 sec default so if yours are 2 sec then the basic settings are off.

on the output node there is a FPS setting and essentially without touching anything else (the frame interpol works as well but you need math tog et it right) by increasing the fps here you will SHORTEN the video to 4 sec (8fps = 6 sec, 10fps = 4 sec, and 6fps will make 8 sec yada yada) but then you are dealing with a slow fps video, and need topazai or adobe premiere to "fix" the entire thing.

My videos are too big for github here, but you might have seen me posting large ones around certain discord model ai channels already. I leave my settings at 8fps and then post process after. 

This is a RAW example - default 6 sec to show you.

https://github.com/user-attachments/assets/c15d2c1c-a9aa-400e-b60b-3286559fbd3b

**Comment by KrakeyMTL** [comment]
I just tested with a fresh install on one of my new servers and can confirm that yes, 6 sec default on clean!

https://github.com/user-attachments/assets/5271b6aa-42d6-4056-8f7d-09e393054825
